upd_script/update_all.sh

```diff
@@ -27,20 +27,65 @@ DEXTER=Dexter
 SCRATCH=Scratch_GUI
 SCRATCH_PATH=$HOME/$DEXTER/$SCRATCH
 
-echo "--> Begin Update."
-echo "--> ======================================="
+feedback() {
+  # first parameter is text to be displayed
+  echo -e "$(tput setaf 3)$1$(tput sgr0)"
```

Comment:
@CleoQc could you document this a little more with another comment or two explaining what's going on.  It looks like you use this quite a bit.